Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls/WrapperLookup.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Forms.UI.Controls
+{
+    /// <summary>
+    /// WrapperLookup is a set of extension methods to extend <see cref="FrameworkElement"/> to make it relatively easy
+    /// to find its associated WindowsXamlHostBaseExt.
+    /// (WPF Interop uses an attached DependencyProperty for this).
+    /// </summary>
+    public static class WrapperLookup
+    {
+        private static readonly IDictionary<FrameworkElement, WindowsXamlHostBaseExt> _controlCollection = new Dictionary<FrameworkElement, WindowsXamlHostBaseExt>();
```

Comment:
If a control is only used on one specific UI element (like a pop-up form), then the form is disposed, the control reference here could not be GC'd because it's referenced by the dictionary. Consider switching to weak references to permit GC

Reply:
I've made the adjustment for this in this branch...  may have to tweak it when we bring in more of the WinForms controls...